### PR TITLE
Get Soft, Get Based

### PR DESCRIPTION
**Pull request by masonicGIT**
**Description**

Bridging SOFT to BASE. 

get-soft-banner

**Tests**

I have not added any tests. I'm sorry father.

**Additional context**

@cfluke-cb 

**Metadata**

- Fixes #[Link to Issue]

### CODE REVIEW

**Comment by cfluke-cb** [comment]
Hi @masonicGIT the image does not seem to be valid. Can you fix?

**Review by masonicGIT — COMMENTED** [review]
Fixing SVG logo

**Comment by therealmccaw** [comment]
Hello @cfluke-cb,

masonicGIT has updated the .svg logo and it appears to be valid now. Thanks for all your help here.

File: data/SOFT/data.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "name": "SoftDAO",
+  "symbol": "SOFT",
+  "decimals": "18",
```
Comment (njokuScript) [inline data/SOFT/data.json:4]: here @masonicGIT

Comment (masonicGIT) [reply data/SOFT/data.json]: @njokuScript !!

**Review by masonicGIT — COMMENTED** [review]
Fixed

**Comment by masonicGIT** [comment]
gentle nudge: @cfluke-cb @njokuScript

**Comment by masonicGIT** [comment]
> @cfluke-cb @njokuScript

slightly firmer nudge @cfluke-cb @njokuScript

**Comment by masonicGIT** [comment]
image

@cfluke-cb @njokuScript

**Comment by masonicGIT** [comment]
> image
> 
> @cfluke-cb @njokuScript

old

@cfluke-cb @njokuScript

**Review by wbnns — APPROVED** [review]
ACK

**Review by annieke — APPROVED** [review]

**Pull request merged**